core/handlers/library/registry.go

```diff
@@ -73,6 +75,8 @@ func InitRegistry(c Config) Registry {
 
 // loadHandlers loads the configured handlers
 func (r *registry) loadHandlers(c Config) {
+	r.authenticationTimeWindow = c.AuthenticationTimeWindow
```

Comment:
we don't have this for expiration check so why is this needed? what am i missing?